### ISSUE TITLE
Screen Reader reads wrong number of issues on cart page

### ISSUE THREAD

**Issue by MichaelBoeser** [SAP/spartacus#14675]
Create cart with two products, product 1 (e.g. CPQ_HOME_THEATER) and product 2 (e.g. WEC_COLOR_TEST). Product 1 should have 1 issue, product 2 should have 2 issues. Tab to resolve issues-link for product 2.  Screen reader reads "1 issue must be resolved" instead of "2 issues must be resolved". possible problem: both links use the same message-id as aria-described by attribute, message-id is not unique
